src/components/EmptyTransactions.tsx

```diff
@@ -16,7 +17,7 @@ const EmptyTransactions = props => (
         </h2>
         <h2 className="subtitle">Need to buy Kinesis?</h2>
         <div>
-          <a>Visit the exchange</a>
+          <Link to="/exchange">Visit the exchange.</Link>
```

Comment:
Maybe we should use the enum for this ? `RootRoutes.exchange`

Reply:
oh yeh I forgot we had that implemented

Reply:
Can cleanup